hx_lti_initializer/templates/hx_lti_initializer/dashboard_view.html

```diff
@@ -27,7 +27,12 @@
         <div class="navigation">
                 <a href="{% url 'hx_lti_initializer:course_admin_hub' %}">Home</a>
         </div>
+        <br>
             <div class="col-md">
+                <div>
+                    <input type="text" id="studentsearch" class="form-control" placeholder="Search for user...">
```

Comment:
@jazahn The way I have the search bar set up right now is that it updates the panels that are shown as the user types, so nothing is actually ever submitted. That said, would that be an accessibility issue?